### PR TITLE
Reverse display name letters for non English names (e.g. Hebrew)

### PR DESCRIPTION
**Pull request by Tal-D**

### CODE REVIEW

File: lib/contact_fixer.rb [hunk 1]
```diff
@@ -4,6 +4,18 @@ def initialize(contacts_api, output)
     @output = output
   end
 
+  def is_non_roman(str)
+    str =~ /[^\w\s!..]/
```
Comment (georgettica) [inline lib/contact_fixer.rb:8]: Maybe make it static?

Comment (georgettica) [reply lib/contact_fixer.rb:8]: Also, add a comment from where you pulled this magic / how does it work

**Review by georgettica — COMMENTED** [review]
Small nits, but overall looks god

File: lib/contact_fixer.rb (hunk 1 above)
Comment (Tal-D) [reply lib/contact_fixer.rb:11]: Done

**Comment by georgettica** [comment]
Lgtm :)
Merging

**Pull request merged**